cpp/ps0000/ps0000solution.cpp

```diff
@@ -1,161 +1,9 @@
-﻿
-#include <iostream>
-#include "hadhod.cpp"
-#include "easylearn.cpp"
+#include <iostream> // 2/2
 using namespace std;
 
-
-
-
-int main()
+int main() // 1/1
 {
-    cout << "Hello, World!";
-    return 1;
+    cout << "Hello, World!\n"; // 3/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [ ] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line